components/nexus-orient/pom.xml

```diff
@@ -62,15 +62,9 @@
     </dependency>
 
     <dependency>
-      <groupId>junit</groupId>
-      <artifactId>junit</artifactId>
-      <optional>true</optional>
-    </dependency>
-
-    <dependency>
       <groupId>org.slf4j</groupId>
       <artifactId>jul-to-slf4j</artifactId>
-      <optional>true</optional>
+      <scope>test</scope>
     </dependency>
```

Comment:
ic, minor downside to this change, is that tests in nexus-orient can't use this rule.  granted tests are a bit light atm, but this does mean if we add more tests we may have to duplicate those rules, or find other ways to do the same thing.  BackupRestoreTrial for example, probably could have been changed to use the persistent rule.

the rules came much later than those, and were never updated.  I wonder if they were updated, if we would still think extracting the rules was a good idea?